src/Slider.jsx

```diff
@@ -667,4 +653,31 @@ class Rheostat extends React.Component {
 Rheostat.propTypes = propTypes;
 Rheostat.defaultProps = defaultProps;
 
-export default Rheostat;
+export const RheostatStyles = ({ color, unit }) => ({
```

Comment:
I'm a little confused about why exactly you are exporting this function. I see that you are using it in the storybook example and in the test, but I don't understand why you need to do it that way instead of exporting the styled component here.